src/utilities/utilities_functions.saved

```diff
@@ -0,0 +1,275 @@
+hypre_SetSpGemmAlgorithm
+hypre_SetSpGemmBinned
+hypre_SetSpGemmRownnzEstimateMethod
+hypre_SetSpGemmRownnzEstimateMultFactor
+hypre_SetSpGemmRownnzEstimateNSamples
+hypre_SetSpGemmUseVendor
+hypre_SetSpMVUseVendor
+hypre_SetSpTransUseVendor
+hypre_SetUseGpuRand
+hypre_SetUserDeviceMalloc
+hypre_SetUserDeviceMfree
+hypre_UnorderedBigIntMapCreate
+hypre_UnorderedBigIntMapDestroy
+hypre_UnorderedBigIntSetCopyToArray
+hypre_UnorderedBigIntSetCreate
+hypre_UnorderedBigIntSetDestroy
+hypre_UnorderedIntMapCreate
+hypre_UnorderedIntMapDestroy
+hypre_UnorderedIntSetCopyToArray
+hypre_UnorderedIntSetCreate
+hypre_UnorderedIntSetDestroy
+hypre_IntArrayCloneDeep
+hypre_IntArrayCloneDeep_v2
+hypre_IntArrayCopy
+hypre_IntArrayCount
+hypre_IntArrayCountHost
+hypre_IntArrayCreate
+hypre_IntArrayDestroy
+hypre_IntArrayInitialize
+hypre_IntArrayInitialize_v2
+hypre_IntArrayInverseMapping
+hypre_IntArrayInverseMappingHost
+hypre_IntArrayMigrate
+hypre_IntArrayPrint
+hypre_IntArrayRead
+hypre_IntArraySetConstantValues
+hypre_IntArraySetConstantValuesHost
+hypre_Log2
+HYPRE_SetGPUMemoryPoolSize
+hypre_Free
+hypre_MAlloc
+hypre_CAlloc
+hypre_CheckMemoryLocation
+hypre_Free
+hypre_GetExecPolicy1
+hypre_GetExecPolicy2
+hypre_GetMemoryLocationName
+hypre_GetPointerLocation
+hypre_MAlloc
+hypre_MemPrefetch
+hypre_Memcpy
+hypre_Memset
+hypre_ReAlloc
+hypre_ReAlloc_v2
+hypre_SetCubMemPoolSize
+hypre_IntArrayMergeOrdered
+hypre_big_merge_sort
+hypre_big_sort_and_create_inverse_map
+hypre_merge_sort
+hypre_sort_and_create_inverse_map
+hypre_union2
+hypre_mm_is_valid
+hypre_mm_read_banner
+hypre_mm_read_mtx_crd_size
+hypre_MPI_Abort
+hypre_MPI_Address
+hypre_MPI_Allgather
+hypre_MPI_Allgatherv
+hypre_MPI_Allreduce
+hypre_MPI_Alltoall
+hypre_MPI_Barrier
+hypre_MPI_Bcast
+hypre_MPI_Comm_create
+hypre_MPI_Comm_dup
+hypre_MPI_Comm_f2c
+hypre_MPI_Comm_free
+hypre_MPI_Comm_group
+hypre_MPI_Comm_rank
+hypre_MPI_Comm_size
+hypre_MPI_Comm_split
+hypre_MPI_Finalize
+hypre_MPI_Gather
+hypre_MPI_Gatherv
+hypre_MPI_Get_count
+hypre_MPI_Group_free
+hypre_MPI_Group_incl
+hypre_MPI_Init
+hypre_MPI_Iprobe
+hypre_MPI_Irecv
+hypre_MPI_Irsend
+hypre_MPI_Isend
+hypre_MPI_Op_create
+hypre_MPI_Op_free
+hypre_MPI_Probe
+hypre_MPI_Recv
+hypre_MPI_Recv_init
+hypre_MPI_Reduce
+hypre_MPI_Request_free
+hypre_MPI_Scan
+hypre_MPI_Scatter
+hypre_MPI_Scatterv
+hypre_MPI_Send
+hypre_MPI_Send_init
+hypre_MPI_Startall
+hypre_MPI_Test
+hypre_MPI_Testall
+hypre_MPI_Type_commit
+hypre_MPI_Type_contiguous
+hypre_MPI_Type_free
+hypre_MPI_Type_hvector
+hypre_MPI_Type_struct
+hypre_MPI_Type_vector
+hypre_MPI_Wait
+hypre_MPI_Waitall
+hypre_MPI_Waitany
+hypre_MPI_Wtick
+hypre_MPI_Wtime
+hypre_GpuProfilingPopRange
+hypre_GpuProfilingPushRange
+hypre_GpuProfilingPushRangeColor
+hypre_prefix_sum
+hypre_prefix_sum_multiple
+hypre_prefix_sum_pair
+hypre_prefix_sum_triple
+free_format
+hypre_ParPrintf
+hypre_fprintf
+hypre_fscanf
+hypre_ndigits
+hypre_printf
+hypre_scanf
+hypre_sprintf
+hypre_sscanf
+new_format
```

Comment:
The `functions.saved` file reveals something interesting. After looking at this list of functions, we can see that some do not start with `hypre_`. This is a note that we should fix them @rfalgout

Reply:
Good points. For now, static functions are not converted. I think that should be fine, but we can discuss this further.